pkg/kubelet/cm/cpumanager/policy_options.go

```diff
@@ -99,3 +112,17 @@ func NewStaticPolicyOptions(policyOptions map[string]string) (StaticPolicyOption
 	}
 	return opts, nil
 }
+
+func ValidateStaticPolicyOptions(opts StaticPolicyOptions, topology *topology.CPUTopology, topologyManager topologymanager.Store) error {
+	if opts.AlignBySocket == true {
+		//1. Not compatible with topology manager single-numa-node policy option.
+		if topologyManager.GetPolicy().Name() == topologymanager.PolicySingleNumaNode {
+			return fmt.Errorf("Topolgy manager Single numa policy is incompatible with CPUManager Align  by socket policy option")
```

Comment:
typos in the string, please also use a `%s` to pull in the actual name of the policy as stored in `topologymanager.PolicySingleNumaNode` instead of writing it out yourself.

Reply:
fixed in latest commit